zebra-state/src/request.rs

```diff
@@ -503,4 +503,13 @@ pub enum ReadRequest {
     ///
     /// Returns a type with found utxos and transaction information.
     UtxosByAddresses(HashSet<transparent::Address>),
+
+    /// Looks up a block hash by height in the current best chain.
+    ///
+    /// Returns
+    ///
+    /// * [`Response::Hash(Some(Hash))`](Response::Hash) if the block is in the best chain;
+    /// * [`Response::Hash(None)`](Response::Hash) otherwise.
+    ///
+    Hash(Height),
```

Comment:
We probably want to call this request `BlockHash`, because we might add a `TransactionHash` request later.

```
    BlockHash(Height),
```

Reply:
Yea, i agree, this was done at https://github.com/ZcashFoundation/zebra/pull/4967/commits/8859808be6b8efcd329da08a35239d404e519700